lib/components/form/batch-settings.tsx

```diff
@@ -143,65 +133,36 @@ class BatchSettings extends Component<Props, State> {
       return
     }
     // Close any expanded panels.
-    this.setState({ expanded: null })
+    setExpanded(null)
 
     // Plan trip.
     routingQuery()
   }
 
-  _updateExpanded = (type: string) => ({
-    expanded: this.state.expanded === type ? null : type
-  })
-
-  _toggleDateTime = () => this.setState(this._updateExpanded('DATE_TIME'))
-
-  _toggleSettings = () => this.setState(this._updateExpanded('SETTINGS'))
-
-  render() {
-    const { activeSearch, config, currentQuery, intl, modeOptions } = this.props
-    const { expanded, selectedModes } = this.state
-    return (
-      <>
-        <ModeButtonsFullWidthContainer className="hidden-lg">
-          <ModeButtonsFullWidth
-            className="flex"
-            modeOptions={modeOptions}
-            onClick={this._onClickMode}
-            selectedModes={selectedModes}
-          />
-        </ModeButtonsFullWidthContainer>
-        <MainSettingsRow>
-          <SettingsPreview
-            aria-label={intl.formatMessage({
-              id: 'components.BatchSettings.settings'
-            })}
-            expanded={expanded === 'SETTINGS'}
-            onClick={this._toggleSettings}
-          >
-            {coreUtils.query.isNotDefaultQuery(currentQuery, config) && (
-              <Dot className="dot" />
-            )}
-            <StyledIconWrapper size="2x">
-              <Cog />
-            </StyledIconWrapper>
-          </SettingsPreview>
-          <StyledDateTimePreview
-            // as='button'
-            expanded={expanded === 'DATE_TIME'}
-            hideButton
-            onClick={this._toggleDateTime}
+  const _updateExpanded = (type: string) => (expanded === type ? null : type)
+
+  const _toggleDateTime = () => setExpanded(_updateExpanded('DATE_TIME'))
```

Comment:
These methods seem superfluous and should be removed. Use the setter from `useState` directly.